qt/aqt/__init__.py

```diff
@@ -50,6 +50,15 @@
 if TYPE_CHECKING:
     import aqt.profiles
 
+from enum import Enum, auto
+
+
+class CallingFunction(Enum):
+    CLAYOUT = auto()
+    MAIN_WINDOW = auto()
+    PREVIWER = auto()
+
```

Comment:
I haven't reviewed the rest of the PR yet, but this feelsunclean to me. The enum is specific to a hook, and doesn't belong in the file responsible for Anki's startup. It would be better placed in either webview.py or the hook code, and with a more specific name like WebviewDidInitContext.

Reply:
I moved it into webview with the new name WebviewDidInitContext.